src/middleware/error-handler.js

```diff
@@ -1,16 +1,21 @@
-const logger = require('../utils/logger');
+const { logger } = require("../utils/logger");
 
-const errorHandler = (
-  err,
-  req,
-  res,
-  next
-) => {
-  logger.error(err.message);
-  if (!err.statusCode) err.statusCode = 500;
-  res.status(err.statusCode).send(err.message);
+const ERROR_CODES = {
+  UNPROCESSABLE_ENTITY: 422,
+  NOT_FOUND: 404,
+  INTERNAL_SERVER_ERROR: 500
+};
+
+const errorHandler = (err, req, res, next) => {
+  console.log("raw", err);
+  const error = JSON.parse(err.message);
```

Comment:
`message` would be a better name for this variable. it's not the error object itself, it's just the message.

Reply:
Even though it comes in the `err.message` property, the resulting object still has `status` and `message` inside it.
If we were to rename this to `message`, we'd end up with
```
res.status(message.status).json(message.message);
```
which seems kinda clunky to me.

Reply:
Fair enough. :)